### DISCUSSION TITLE
deprecate max support?

### DISCUSSION THREAD

**Discussion by dromer** [Wasted-Audio/hvcc#25] [Ideas]
There are some remnants that suggest that max support was once there, but afaik this is currently not actually working and max users have `~gen` anyway.

Better drop any mention/support and just focus on Pd?

See: [hvcc/interpreters/max2hv](https://github.com/Wasted-Audio/hvcc/tree/develop/hvcc/interpreters/max2hv)

**Comment by dgbillotte** [comment]
I haven't heard any calls for max support here or over in the rebel tech forums since this was posted 14 months ago. All of the max related posts I've seen seem to be using ~gen for the OWL platform. I don't know about Bela, Daisy, ...

I vote to deprecate it.

**Reply by dromer** [reply]
Yeah I think everyone knows that heavy/hvcc is a compiler for pure data patches, so supporting Max/MSP just puts extra pressure/strain that I don't think is very realistic to do, right?

Unless someone steps forward and says "no wait! this works and is very important to me!" this should for sure just be removed before any kind of `1.0` release, at least. :)
(and maybe even sooner)

**Comment by dromer** [comment]
Well, it is done.

`max2hv` has been purged and all mentions removed from the docs (afaict).
